lua/hbac/init.lua

```diff
@@ -3,12 +3,14 @@ local id = vim.api.nvim_create_augroup("hbac", {
 })
 
 local M = {
-	persistant_buffers = {},
+	pinned_buffers = {},
 }
 
-M.persist_buffer = function(bufnr)
+local toggle_pin = function(bufnr, action)
 	bufnr = bufnr or vim.api.nvim_get_current_buf()
-	M.persistant_buffers[bufnr] = true
+	action = action or "pin"
+	M.pinned_buffers[bufnr] = action ~= "unpin"
```

Comment:
```suggestion
	M.pinned_buffers[bufnr] = action == "pin"
```

`~= "unpin"` is an artifact from the buggy prior commit, will fix if you decide this refactor is acceptable